src/Sinch/SinchClient.cs

```diff
@@ -271,27 +266,22 @@ public ISinchAuth Auth
 
         /// <inheritdoc />
         public ISinchVoiceClient Voice(string appKey, string appSecret,
-            CallingRegion? callingRegion = default,
-            AuthStrategy authStrategy = AuthStrategy.ApplicationSign)
+            CallingRegion? callingRegion = default)
```

Comment:
Not related to this PR, but:
- `SmsHostingRegion` for SMS (https://github.com/sinch/sinch-sdk-dotnet/blob/main/src/Sinch/SinchOptions.cs#L32)
- `ConversationRegion` for Conversation (https://github.com/sinch/sinch-sdk-dotnet/blob/main/src/Sinch/SinchOptions.cs#L38)
- `CallingRegion` for Voice.

`SmsRegion` & `ConversationRegion` & `VoiceRegion` seems better to provide same naming convention inside this one but also across SDKs